src/networking.md

```diff
@@ -17,6 +17,12 @@ blockchain](./blockchain-runtime-interface.md#runtime-functions) to connect to m
 to the [mixnet topology](./topology.md). If no external addresses have been published for a
 mixnode, or none of them work, nodes should attempt to discover addresses using the libp2p DHT.
 
+// Here I am not sure, do we have the Dht? or if we are talking about discovery dht, we should name
+// it. Also it implies that we default to the common validator address (an implementor could make all mixnet
+// in a different address (different node than the validating one. Even if not doable with current implementation, this sounds like easy double signing slashing for wrongly configured validators though, also may need a switch in the registration process to register on a single node, may be some smarter conf where we call the transaction to the validator node from the mixnet node to avoid duplicating key, but that is more work)).
```

Comment:
I think the mixnode being different to the validator node can be supported, particularly with the registration change that's coming soon, though it's not really supported with the current node code. The mixnode peer ID and addresses are contained in the registration transaction and need not be those of the validator node. All the validator node would need to do is sign these transactions (one per session).

Reply:
Let s keep it in mind for later then. But yes would be very nice I think.